### PR TITLE
Add Viaq installation guide to the oVirt metrics store repo

### PR DESCRIPTION
**Pull request by sradco**
This adds the viaq installation to the oVirt repo, So we can remove the steps that were already automated.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sradco

This pull request needs review by: @mykaul

### CODE REVIEW

**Comment by sradco** [comment]
@richm Whould appreciate your review of my change.

File: source/develop/release-management/features/metrics/setting-up-viaq-logging.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+      # semanage fcontext -a -t svirt_sandbox_file_t "/var/lib/elasticsearch(/.*)?"
+        
+      # restorecon -R -v /var/lib/elasticsearch
+
+
+Installing ViaQ Packages
+------------------------
+
+These instructions and config files are for an all-in-one, single machine, run
+ansible on the same machine you are installing ViaQ on.
```
Comment (richm) [inline source/develop/release-management/features/metrics/setting-up-viaq-logging.md]: This is the same as the next paragraph.  You can remove it.

File: source/develop/release-management/features/metrics/setting-up-viaq-logging.md [hunk 2]
```diff
@@ -88,11 +88,18 @@ configured to use persistence.
...
+- Create the directory
+  
+      # mkdir -p /var/lib/elasticsearch
+  
 - Change the group ownership to the value of your
   `openshift_logging_elasticsearch_storage_group` parameter (default `65534`)
-  e.g. `chgrp 65534 /var/lib/elasticsearch`
-- make this directory writable by the group `chmod -R g+w /var/lib/elasticsearch`
+  
+      # chgrp 65534 /var/lib/elasticsearch`
```
Comment (richm) [inline source/develop/release-management/features/metrics/setting-up-viaq-logging.md]: I think you need a blank line after this line

Comment (sradco) [reply source/develop/release-management/features/metrics/setting-up-viaq-logging.md]: Done. Thanks.

Comment (rollandf) [reply source/develop/release-management/features/metrics/setting-up-viaq-logging.md]: @sradco I think you are missing the headers, like these for example:
 ---
 title: oVirt Metrics Store - Installation Guide
 category: feature
 authors: sradco
 feature_name: oVirt Metrics Store Installation
 feature_modules: engine
 feature_status: In Development
---

Please add them, and we will see if the CI pass

**Pull request merged**